src/features/common/data/indexed-db.ts

```diff
@@ -24,7 +26,8 @@ interface LoraDBSchemaV2 extends DBSchema {
 }
 
 export async function getDbConnection(networkId: string) {
-  return await openDB<LoraDBSchemaV2>(networkId, 2, {
+  const genesisHash = await dataStore.get(genesisHashAtom)
```

Comment:
Accessing like this means that this function will never be updated if the atom is updated. It means that this function relies on the behaviour of the provider to get the correct value.

I'll push a change which matches what we do for networkId.

Reply:
I can't think of any other options, so I think this is fine.